src/main/java/org/jabref/gui/actions/LookupIdentifierAction.java

```diff
@@ -97,8 +99,9 @@ private String lookupIdentifiers() {
                 if (fieldChange.isPresent()) {
                     namedCompound.addEdit(new UndoableFieldChange(fieldChange.get()));
                     foundCount++;
-                    frame.output(Localization.lang("Looking up %0... - entry %1 out of %2 - found %3",
-                            Integer.toString(count), totalCount, Integer.toString(foundCount)));
+                    final String nextStatusMessage = Localization.lang("Looking up %0... - entry %1 out of %2 - found %3",
+                            fetcher.getIdentifierName(), Integer.toString(count), totalCount, Integer.toString(foundCount));
+                    DefaultTaskExecutor.runInJavaFXThread(() -> frame.getDialogService().notify(nextStatusMessage));
```

Comment:
lookupIdentifiers is executed in a background thread and in order to provide feedback to the user, i used a workaround here and explicitly called DefaultTaskExecutor.runInJavaFXThread
However, i would suggest to completely remove the notifications here, because they do not provide much value